Formula/gnunet.rb

```diff
@@ -0,0 +1,36 @@
+class Gnunet < Formula
+  desc "GNUnet"
+  homepage "https://gnunet.org/"
+  url "https://ftp.gnu.org/gnu/gnunet/gnunet-0.11.0.tar.gz"
+  sha256 "b7477a3c3b0d5e8a013685dc208cfb4ccee4145f8668faa8eb5b382af36c7e9a"
+
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+  depends_on "gnutls"
+  depends_on "jansson"
+  depends_on "libextractor"
+  depends_on "libgcrypt"
+  depends_on "libidn2"
+  depends_on "libmicrohttpd"
+  depends_on "libmpc"
+  depends_on "libunistring"
+  depends_on "unbound"
+
+  def install
+    args = %W[
+      --disable-documentation
+      --prefix=#{prefix}
+    ]
+
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  def post_install
+    chmod "+x", prefix/"bin/gnunet-qr.py"
```

Comment:
Please do the `chmod` in `install` instead.

Reply:
I think gnunet removes the executable flag from the script on "make install".
Hence my chmod has no effect in "install".
So this is not the fault of the formula, it is just something gnunet does atm.

Reply:
Do it in the `install` method, but after `make install`.

Reply:
Done.

Reply:
I tried a variety of ways to make this error go away in the audit locally. It doesn't work for me in ```install```. I reverted it to ```post_install``` for now.

Reply:
I manually checked:
in ```install``` after the make install I can set the +x bit. But homebrew does something _after_ install which removes the bit again. This is independent of the formula.
The build servers fail because when building a bottle the ```post_install``` is not executed. I do not know how to fix this.

Reply:
Anyway. I think GNUnet is doing something weird here. A python script (without shebang) should not be installed in bin/.
For now (44bde0a92..583899528) I move it to pkgshare in order for the build to pass.
In addition, I will file an upstream bug.

Reply:
Builds/checks seem to pass now. My last commit moves the python script to ```pkgshare``` and patches the ```gnunet-qr``` script which calls the python script in order to find it in the new location.